src/app/components/boards/boards.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { RestApiService } from 'src/app/services/restapi.service';
+import { TranslateService } from '@ngx-translate/core';
+import { OpenBoardService } from 'src/app/services/open-board.service';
+import { SwalService } from 'src/app/services/swal.service';
+import { board } from 'src/app/interfaces/interfaces';
+
+@Component({
+    selector: 'app-boards',
+    templateUrl: './boards.component.html',
+    styleUrls: ['./boards.component.css']
```

Comment:
According to best practices, it better to use onPush strategy ( changeDetection: ChangeDetectionStrategy.OnPush) for each component. That helps you to prevent performance issue with application.